tutorials/text/text_props.py

```diff
@@ -204,14 +212,27 @@
 # Korean, or Japanese.
 #
 # To set the default font to be one that supports the code points you
-# need, prepend the font name to ``'font.family'`` or the desired alias
-# lists ::
+# need, prepend the font name to ``'font.family'`` (recommended), or to the
+# desired alias lists (not recommended). ::
+#
+#    # recommended
+#    matplotlib.rcParams['font.family'] = ['Source Han Sans TW', 'sans-serif']
+#
+#    # not recommended
+#    matplotlib.rcParams['font.family'] = ['sans-serif']
+#    matplotlib.rcParams['sans-serif'] = ['Source Han Sans TW', ...]
+#
+# This is because these generic family alias lists should ideally contain fonts
+# that are shipped alongside Matplotlib, which work across all installations,
+# unlike user-defined fonts which have no such guarantee.
 #
-#    matplotlib.rcParams['font.sans-serif'] = ['Source Han Sans TW', 'sans-serif']
+# A good practice when setting custom font families is to append
+# a generic-family to the font-family list as a last resort; when none of the
+# families contain the required glyph.
```

Comment:
But we do not currently do per-glyph fallback, only "can we find this font?" fallback.

Reply:
That's true, but even in-general usecases, this holds true.
(would be 100% true after we've accomplished true fallback)

Reply:
We should hold off on including "...contain the required glyph." until it is all the way true ;)

It is my understanding that these values eventually make it to https://github.com/matplotlib/matplotlib/blob/a3fbf1876830a89087842ea779515e0d099c885d/lib/matplotlib/font_manager.py#L1130-L1158 which only takes into account the name (not the pages supported).

Reply:
That also makes sense, let me remove that line 👍🏼